tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "es2015",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": false,
+    "moduleResolution": "node",
+    "stripInternal": true,
+    "preserveConstEnums": true,
+    "listFiles": true,
+    "declaration": true,
+    "removeComments": true,
+    "lib": ["es2015", "dom"]
```

Comment:
@atsu85 `lib` is an official setting of TypeScript 2.0 beta, which we're using only because it makes generating both `es2015` and `es5` easier without including external shims for `es5`. Since we're only using TS to generate typings, we don't need to worry about the fact that this is a beta. Relevant issue @TypeScript: https://github.com/Microsoft/TypeScript/issues/6974